public/app/features/alerting/unified/mocks/server/handlers.ts

```diff
@@ -4,7 +4,60 @@
 
 import { HttpResponse, http } from 'msw';
 
+import { mockAlertmanagerAlert, mockSilences } from 'app/features/alerting/unified/mocks';
 import { defaultAlertmanagerChoiceResponse } from 'app/features/alerting/unified/mocks/alertmanagerApi';
+import { AlertState } from 'app/plugins/datasource/alertmanager/types';
+
+///////////////////
+// Alertmanagers //
+///////////////////
 
 export const alertmanagerChoiceHandler = (response = defaultAlertmanagerChoiceResponse) =>
   http.get('/api/v1/ngalert', () => HttpResponse.json(response));
+
+const alertmanagerAlertsListHandler = () =>
+  http.get('/api/alertmanager/:datasourceUid/api/v2/alerts', () =>
+    HttpResponse.json([
+      mockAlertmanagerAlert({
+        labels: { foo: 'bar', buzz: 'bazz' },
+        status: { state: AlertState.Suppressed, silencedBy: ['12345'], inhibitedBy: [] },
+      }),
+      mockAlertmanagerAlert({
+        labels: { foo: 'bar', buzz: 'bazz' },
+        status: { state: AlertState.Suppressed, silencedBy: ['12345'], inhibitedBy: [] },
+      }),
+    ])
+  );
+
```

Comment:
I didn't generalise this here, not sure what we want the mock API to return as a baseline - any ideas?

Reply:
In the best scenario this should be tied to datasources setup. 
404, at least in theory should be pretty rare (only Cortex responds this way)
For Prometheus and Mimir datasources we should provide valid JSON. GMA doesn't send this request